frontend/src/components/cards/age-category/age-category.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import ReactEcharts from 'echarts-for-react';
+import { Card } from '../../layout/card/card';
+import { formatDate } from '../../../utils/date';
+
+export const EMBED_PATH_AGE_CATEGORY = 'categorie-varsta';
+export class AgeCategory extends React.PureComponent {
+  getChartOptions = (state) => {
+    const { ageCategories, dateStrings } = state;
+    const series =
+      ageCategories &&
+      Object.entries(ageCategories).map(([key, value]) => ({
+        data: value,
+        type: 'line',
+        smooth: true,
+        name: key,
+      }));
+    return {
+      grid: {
+        left: 40,
+        top: 40,
+        right: 0,
+        bottom: 20,
+      },
+      legend: { type: 'scroll' },
+      icon: 'roundRect',
+      xAxis: {
+        type: 'category',
+        data: dateStrings,
+      },
+      yAxis: {},
+      tooltip: {
+        trigger: 'axis',
+      },
+      dataZoom: [
+        {
+          type: 'slider',
+          start: 60,
+          end: 100,
+        },
+      ],
+      series,
+    };
+  };
+
+  render() {
+    const { title, state } = this.props;
+    const { isLoaded, error, lastUpdatedOn, stale } = state;
+    return (
+      <Card
+        isLoaded={isLoaded}
+        error={error}
+        title={title}
+        subtitle={`Ultima actualizare: ${formatDate(lastUpdatedOn)}`}
+        isStale={stale}
+        embedPath={EMBED_PATH_AGE_CATEGORY}
+      >
+        <ReactEcharts
```

Comment:
Can we change this to:
```
        <ReactEcharts
          style={{
            height: '470px',
            width: '100%',
          }}
          id="age-category"
          option={this.getChartOptions(state)}
          theme="light"
        />
```
To make the graph a little taller? (it will be just as tall as `CasesPerDayCard`)